package.json

```diff
@@ -9,7 +9,7 @@
     "build:all": "yarn build:cjs && yarn build:es6",
     "build:cjs": "webpack --config build-settings/webpack.config.js --display-modules && node utils/package-flow-types.js",
     "build:es6": "rollup -c build-settings/rollup.config.js",
-    "build:storybook": "build-storybook",
+    "build-storybook": "build-storybook",
```

Comment:
Chromatic expects this to be the name of the build script.  I'm sure there's a way to configure this.  I just wanted to get things working first before making that change.